docs/spec.md

```diff
@@ -538,6 +561,20 @@ All integer types explicitly specified (uint32, uint64, etc.) in this section ar
 | _encoding length_ | char[]  | encoding, same encoding that was used for the request |
 | remaining bytes   | uint8[] | response payload                                      |
 
+### Asset
+
+- Response to a [fetch asset](#fetch-asset) request if the asset was found
+- Only supported if the server previously declared the `assets` [capability](#server-info).
+
+| Bytes               | Type    | Description                                                                                            |
+| ------------------- | ------- | ------------------------------------------------------------------------------------------------------ |
+| 1                   | opcode  | 0x04                                                                                                   |
+| 4                   | uint32  | request id, as given in the corresponding [request](#fetch-asset)                                      |
+| 8                   | uint64  | Last modified time of the asset (nanoseconds)                                                          |
+| 4                   | uint32  | media type length                                                                                      |
+| _media type length_ | char[]  | [media type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) of the asset |
+| remaining bytes     | uint8[] | asset data (file contents)                                                                             |
+
```

Comment:
In case the asset can't be fetched (e.g. does not exist), should we sent a special response? Right now that's not the case, and the client would have to determine itself (e.g. through a timeout) if the asset failed to fetch